docs/optimization/improve-parallelism/dataflow.md

```diff
@@ -1,8 +1,8 @@
 ### Enabling dataflow parallelism
 
-This guide teaches what data parallelism is and how it can improve the execution time of Concrete circuits.
+This guide explains data parallelism and how it can improve the execution time of **Concrete** circuits.
 
-Dataflow parallelism is a great feature, especially when the circuit is doing a lot of scalar operations.
+Dataflow parallelism particularly useful when the circuit performs a lot of scalar operations. 
```

Comment:
Dataflow parallelism is particularly useful

Reply:
Thx